meta-mender-qemu/scripts/mender-qemu

```diff
@@ -123,8 +123,15 @@ $QEMU_SYSTEM --version || exit 1
 
 echo "--- starting qemu"
 
+USE_KVM=' -enable-kvm ""'
```

Comment:
I believe this kind of expansion won't work, string quoting is done before variable expansions, so you get the literal `""` value instead.

Reply:
It works :)